gradle/dependencies.gradle

```diff
@@ -20,7 +20,8 @@ def versions = [
     dokka: '0.9.17',
     errorProne: '2.3.2',
     errorPronePlugin: '0.0.13',
-    kotlin: '1.3.0'
+    kotlin: '1.3.0',
+    lint: '26.2.1'
```

Comment:
Let's use the latest - `26.3.0-beta04`. We can mark the min api in the registry via `minApi = 2`